### PR TITLE
add: `endlessh-git`

### PR DESCRIPTION
**Pull request by Henryws**
## Progress
- [x] Edit `packagelist`
- [x] Add initial pacscript
- [ ] Contact devs: https://github.com/skeeto/endlessh/issues/76
- [ ] Add `maintainer` to pacscript

### CODE REVIEW

**Review by wizard-28 — CHANGES_REQUESTED** [review]
1. Add `gives` & `breaks`.
2. Use `pkgdir` instead.
3. Install `UNILICENSE` & `README.md`
4. Install the sample config file (download it from this repository)

File: packages/endlessh-git/endlessh-git.pacscript [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+pkgdir="/usr/src/pacstall/${name}"
+url="https://github.com/skeeto/endlessh.git"
+description="SSH tarpit that very slowly sends an endless, random SSH banner"
+pkgver() {
+	git ls-remote "${url}" master | cut -f1
+}
+version="$(pkgver)"
+
+prepare() {
+	wget -q https://raw.githubusercontent.com/pacstall/pacstall-programs/master/packages/endlessh-git/sample-config
```
Comment (wizard-28) [inline packages/endlessh-git/endlessh-git.pacscript:13]: Verify checksum

**Review by wizard-28 — CHANGES_REQUESTED** [review]

File: packages/endlessh-git/endlessh-git.pacscript [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+description="SSH tarpit that very slowly sends an endless, random SSH banner"
+pkgver() {
+  git ls-remote "${url}" master | cut -f1
+}
+version="$(pkgver)"
+
+prepare() {
+  # sha256sum f92a2341103a7c6243d7cbc32a3400e0f01ecd5fae12a0731029f6c50ba9f451
+  wget -q https://raw.githubusercontent.com/pacstall/pacstall-programs/master/packages/endlessh-git/sample-config
+  if ! [[ "$(sha256sum sample-config)" == "f92a2341103a7c6243d7cbc32a3400e0f01ecd5fae12a0731029f6c50ba9f451  sample-config" ]]; then
```
Comment (wizard-28) [inline packages/endlessh-git/endlessh-git.pacscript:15]:
```suggestion
  if [[ "$(sha256sum sample-config | cut -d' ' -f 1)" != "f92a2341103a7c6243d7cbc32a3400e0f01ecd5fae12a0731029f6c50ba9f451" ]]; then
```

**Review by wizard-28 — CHANGES_REQUESTED** [review]

File: packages/endlessh-git/endlessh-git.pacscript [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+pkgdir="/usr/src/pacstall/${name}"
+url="https://github.com/skeeto/endlessh.git"
+description="SSH tarpit that very slowly sends an endless, random SSH banner"
+pkgver() {
+  git ls-remote "${url}" master | cut -f1
+}
+version="$(pkgver)"
+
+prepare() {
+  # sha256sum f92a2341103a7c6243d7cbc32a3400e0f01ecd5fae12a0731029f6c50ba9f451
```
Comment (wizard-28) [inline packages/endlessh-git/endlessh-git.pacscript:13]: What's this comment for? I think a comment saying that it's downloading the sample config would be more helpful.

File: packages/endlessh-git/endlessh-git.pacscript [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+}
+
+build() {
+  make -j"$(nproc)"
+}
+
+install() {
+  sudo make install DESTDIR="${pkgdir}"
+  sudo install -Dm644 README.md -t "${pkgdir}/usr/share/doc/${name}"
+  sudo install -Dm644 UNLICENSE -t "${pkgdir}/usr/share/licenses/${name}"
```
Comment (wizard-28) [inline packages/endlessh-git/endlessh-git.pacscript:28]: Remove the empty directories in `removescript()`

**Review by wizard-28 — CHANGES_REQUESTED** [review]

**Comment by Henryws** [comment]
Messed up the git history accidentally, remaking the PR

**Pull request closed without merging**